cmd/koord-manager/main.go

```diff
@@ -101,6 +111,9 @@ func main() {
 	flag.BoolVar(&enablePprof, "enable-pprof", true, "Enable pprof for controller manager.")
 	flag.StringVar(&pprofAddr, "pprof-addr", ":8090", "The address the pprof binds to.")
 	flag.StringVar(&syncPeriodStr, "sync-period", "", "Determines the minimum frequency at which watched resources are reconciled.")
+	pflag.StringSliceVar(&controllers, "controllers", allControllers, fmt.Sprintf("A list of controllers to enable. '*' enables all on-by-default controllers,"+
```

Comment:
We still have a small number of controllers inside, which are controlled by feature-gate. Then follow this to modify?